qemu/tests/cfg/bridge_mirror.cfg

```diff
@@ -19,16 +18,16 @@
     image_snapshot = yes
     login_timeout = 720
     stop_NM_cmd = "systemctl stop NetworkManager"
-    stop_firewall_cmd = "systemctl stop firewalld.service"
-    enable_promisc_cmd = "ifconfig ${private_bridge} promisc up"
-    tc_qdisc = "/sbin/tc qdisc"
-    tc_filter = "/sbin/tc filter"
+    stop_firewall_cmd = "systemctl stop firewalld"
```

Comment:
Do you want to use `utils.service()`?

Reply:
`from avocado.utils import service`?
Looks like it only support host.

Reply:
OK, ignore me~